src/renderer/components/repositoryListView/repositoryList.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+	<div style="width: 100%">
+		<t-scrollbar v-if="getAllRepository.length > 0" height="300px">
+			<div style="padding: 1rem">
+				<div v-if="getAllRepository.length > 0">
```

Comment:
Can you remove this `v-if` statement? It's already defined above.

Reply:
@CodingCreate101 can you just work on this review and then we can merge into https://github.com/gitthermal/thermal/pull/3578